### PR TITLE
Paig SecureChat added

### PR DESCRIPTION
**Pull request by SiddheshwarKamble**
## Change Description

Paig SecureChat

## Issue reference

This PR fixes issue #65

## Checklist

- [x] I have reviewed the [contribution guidelines](https://github.com/privacera/paig/blob/main/docs/CONTRIBUTING.md)
- [x] My code includes unit tests
- [x] All unit tests and lint checks pass locally
- [x] My PR contains documentation updates / additions if required

### CODE REVIEW

**Review by pravin-bansod — APPROVED** [review]

File: paig-securechat/README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Privacera Securechat
+
+Secure Chat is a conversational AI chat bot .
+Secure chat allows users to create  conversations with an AI chat bot which can optionally  be governed by Privacera AI Governance Services(PAIG).
```
Comment (adityasinght) [inline paig-securechat/README.md:4]: governed by Privacera AI Governance Services(PAIG) => governed by PAIG

File: paig-securechat/notebooks/PaigSecureChat.ipynb
Comment (adityasinght) [inline paig-securechat/notebooks/PaigSecureChat.ipynb:1]: Should notebooks be merged with root folder notebook folder?

File: .github/workflows/paig-securechat-release.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        echo "Extracted Version: $VERSION"
+        echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+    - name: Build and push
+      uses: docker/build-push-action@v6
+      with:
+        context: .
+        file: ./paig-securechat/docker/Dockerfile
+        push: true
+        tags: ${{ secrets.DOCKERHUB_USERNAME }}/user_manager:${{ env.VERSION }}
```
Comment (pravin-bansod) [inline .github/workflows/paig-securechat-release.yml:103]: user_manager - paig-securechat

Comment (pravin-bansod) [reply .github/workflows/paig-securechat-release.yml:103]: privacera/paig-securechat:${{ env.VERSION }}

**Review by adityasinght — APPROVED** [review]

**Review by pravin-bansod — APPROVED** [review]

**Pull request merged**